src/notebooklm/_notebooks.py

```
@@ -67,11 +71,71 @@ async def create(self, title: str) -> Notebook:
+                "leaving original RPC error unchanged",
+                exc_info=True,
+            )
+            return
+
+        notebook_limit = account_limits.notebook_limit
+        if notebook_limit is None:
+            return
+
+        try:
+            notebooks = await self.list()
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)